examples/policies/sample-policy.yaml

```diff
@@ -1,12 +1,17 @@
+#   \\
+#  \\\\ SPIKE: Secure your secrets with SPIFFE. — https://spike.ist/
+# \\\\\\
+
 # Example policy configuration for spike policy apply --file
-# This demonstrates how to define a policy using YAML instead of command line flags
+# This demonstrates how to define a policy using YAML instead of command line
+# flags
 
 # Policy name - must be unique within the system
 name: "web-service-policy"
 
 # SPIFFE ID pattern for workload matching
 # This pattern determines which workloads this policy applies to
-spiffeid: "spiffe://example.org/web-service/*"
+spiffeid: "spiffe://example.org/web-service/"
```

Comment:
Removed trailing asterisk from policy files and examples since it is misleading.

These are not path globs, and the meaning of the asterisk is "repeat the last / 0-infinity times".

What might be a better fit there could be something like:

`"^spiffe://example\.org/web-service/.*$`
( start with "spiffe://example.org/web-serice/" followed by one or many characters )